packages/cadl-python/CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 # Release History
 
-## 2022-12-16 - 0.4.15
+## 2023-01-XX - 0.4.15
```

Comment:
`0.4.15` is not released yet: https://www.npmjs.com/package/@azure-tools/cadl-python?activeTab=versions